<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot_Options</class>
 <widget class="QWidget" name="Plot_Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBox_log_y2">
     <property name="text">
      <string>Y-Axis 2 logarithmic</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBox_use_abs_x">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Use Absolute</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox_log_x">
     <property name="text">
      <string>X-Axis logarithmic</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox_use_abs_y">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Use Absolute</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox_log_y">
     <property name="text">
      <string>Y-Axis logarithmic</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBox_use_abs_y2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Use Absolute</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
